Worktez/worktez
# feat(Milestone):Created Milestonecomponent

### Functionality:
Created MilestoneComponent

### Solution:
Users can create a milestone or plan and add tasks to it

### Risk level:

### How to test:
Go to milestones and create a milestone
After creating a milestone create a task with the milestone id
Click on the milestone card to see the milestone Details
